site/templates/footer/app-badges.jet

```diff
@@ -3,7 +3,7 @@
 {{block appBadges()}}
   {{app_link_ios := config("app_link_ios")}}
   {{app_link_android := config("app_link_android")}}
-  {{if config("app_link_ios") != "" || config("app_link_android") != ""}}
+  {{if config("app_link_ios") == "" || config("app_link_android") == ""}}
```

Comment:
Is this right? If the app links are blank they shouldn't be shown should it?

Reply:
Oh snap!  Good catch... no, I was fiddling that for testing.